src/app/series/series.component.ts

```diff
@@ -37,11 +37,20 @@ export class SeriesComponent implements OnInit {
 
   loadSeries() {
     if (this.id) {
-      this.cms.auth.follow('prx:series', {id: this.id}).subscribe(s => {
-        s.follow('prx:account').subscribe(a => {
-          this.setSeries(a, s);
-        });
-      });
+      this.cms.auth.follow('prx:series', {id: this.id}).subscribe(
+        s => {
+          s.follow('prx:account').subscribe(a => {
+            this.setSeries(a, s);
+          });
+        },
+        err => {
+          if (err.status === 404 && err.name === 'HalHttpError') {
```

Comment:
`err instanceof HalHttpError` does not work -- not sure why! But looks like we've run into that before --
 https://github.com/PRX/publish.prx.org/blob/master/src/app/story/directives/metrics-downloads.component.ts#L85